source/incqueryserver-jupyter/iqs_jupyter/analysis_extensions.py

```diff
@@ -19,7 +19,7 @@
 
 import html
 
-import iqs_client
+from iqs_client import models as iqs_client
```

Comment:
`iqs_client` is the import of the root module, do not use the same alias for the `models` submodule